.github/workflows/node.yml

```diff
@@ -0,0 +1,28 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [master]
```

Comment:
`[ $default-branch ]` instead of master ? btw the default branch on this repo is `main`